pkg/operator/operator/workload_logging.go

```diff
@@ -42,6 +49,168 @@ const (
+	Partition string
+	Region    string
+	LogGroup  string
+	APIName   string
+}
+
+type completedJobLogURLTemplateArgs struct {
+	Partition string
+	Region    string
+	StartTime string
+	EndTime   string
+	LogGroup  string
+	APIName   string
+	JobID     string
+}
+
+type inProgressJobLogURLTemplateArgs struct {
+	Partition string
+	Region    string
+	LogGroup  string
+	APIName   string
+	JobID     string
+}
+
+func completedBatchJobLogsURL(args completedJobLogURLTemplateArgs) (string, error) {
+	buf := &bytes.Buffer{}
+	err := _completedJobLogURLTemplate.Execute(buf, args)
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(buf.String()), nil
+}
+
+func inProgressBatchJobLogsURL(args inProgressJobLogURLTemplateArgs) (string, error) {
+	buf := &bytes.Buffer{}
+	err := _inProgressJobLogsURLTemplate.Execute(buf, args)
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(buf.String()), nil
+}
+
+func APILogURL(api spec.API) (string, error) {
+	partition := "aws.amazon"
+	region := config.ClusterConfig.Region
+	if awslib.PartitionFromRegion(region) == "aws-us-gov" {
+		partition = "amazonaws-us-gov"
+	}
+	logGroup := config.ClusterConfig.ClusterName
+
+	args := apiLogURLTemplateArgs{
+		Partition: partition,
+		Region:    region,
+		LogGroup:  logGroup,
+		APIName:   api.Name,
+	}
+
+	buf := &bytes.Buffer{}
+	err := _apiLogURLTemplate.Execute(buf, args)
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(buf.String()), nil
+}
+
+func BatchJobLogURL(apiName string, jobStatus status.BatchJobStatus) (string, error) {
+	partition := "aws.amazon"
+	region := config.ClusterConfig.Region
+	if awslib.PartitionFromRegion(region) == "aws-us-gov" {
+		partition = "amazonaws-us-gov"
+	}
+	logGroup := config.ClusterConfig.ClusterName
+
+	if jobStatus.EndTime != nil {
+		endTime := *jobStatus.EndTime
+		endTime = endTime.Add(60 * time.Second)
+		return completedBatchJobLogsURL(completedJobLogURLTemplateArgs{
+			Partition: partition,
+			Region:    region,
+			StartTime: timeString(jobStatus.StartTime),
+			EndTime:   timeString(endTime),
+			LogGroup:  logGroup,
+			APIName:   apiName,
+			JobID:     jobStatus.ID,
+		})
+	}
+	return inProgressBatchJobLogsURL(inProgressJobLogURLTemplateArgs{
+		Partition: partition,
+		Region:    region,
+		LogGroup:  logGroup,
+		APIName:   apiName,
+		JobID:     jobStatus.ID,
+	})
+}
+
+func TaskJobLogURL(apiName string, jobStatus status.TaskJobStatus) (string, error) {
+	partition := "aws.amazon"
+	region := config.ClusterConfig.Region
+	if awslib.PartitionFromRegion(region) == "aws-us-gov" {
+		partition = "amazonaws-us-gov"
+	}
+	logGroup := config.ClusterConfig.ClusterName
+	if jobStatus.EndTime != nil {
+		endTime := *jobStatus.EndTime
+		endTime = endTime.Add(60 * time.Second)
+		return completedBatchJobLogsURL(completedJobLogURLTemplateArgs{
+			Partition: partition,
+			Region:    region,
+			StartTime: timeString(jobStatus.StartTime),
+			EndTime:   timeString(endTime),
```

Comment:
we can discuss live, I was just wondering why the start and end times are necessary